pkgs/regen.nix

```diff
@@ -1,11 +1,8 @@
-let
-  localLib = import ../lib.nix;
-in
 { system ? builtins.currentSystem
 , config ? {}
-, iohkPkgs ? import ../. { inherit config system; }
-, pkgs ? iohkPkgs.pkgs
-, nixpkgsPath ? localLib.iohkNix.nixpkgs
+, iohkNix ? (import ../lib.nix { inherit config system; }).iohkNix
```

Comment:
This makes this more obviously just dependent on iohkNix (we should really move it into there).